### DISCUSSION TITLE
EIA-860 Schedule 7?

### DISCUSSION THREAD

**Discussion by grgmiller** [catalyst-cooperative/pudl#3439] [Help Me!]
In the EIA-860 instructions, it indicates that respondents can fill in details about "other" responses in schedule 7. However, it does not appear that schedule 7 data is published with the main EIA-860 data publication. For example:
<img width="273" alt="image" src="https://github.com/catalyst-cooperative/pudl/assets/45949268/ba300a83-cba1-45cb-a805-15a63093d278">

Does anyone know if EIA publishes the schedule 7 responses and where they could be found?

**Comment by zaneselvans** [comment]
Huh, there's a bunch of stuff pointing to Schedule 7 [in the instructions](https://www.eia.gov/survey/form/eia_860/instructions.pdf). It seems like it's just an unstructured "comments" section where you can explain things that don't fit elsewhere. I don't remember actually seeing the contents. Might be worth emailing EIA to see if the comments are available somewhere. I suspect it would be at least a little painful to extract as structured data.

**Reply by grgmiller** [reply]
No worries, just thought I'd check - I figured that if anyone knew you all probably would.
